tests/test_food_store_coverage.py

```
@@ -345,3 +377,21 @@ def test_nutrients_for_missing_nutrient_keys(self, mock_get_food):
         assert result["Mg_mg"] == 0.0
         # А присутствующие нутриенты должны быть учтены
         assert result["Fe_mg"] == 1.0
+
+    def test_nutrients_for_negative_per_g(self):
+        """Тест nutrients_for с отрицательным per_g (строка 107)"""
```

Ruff (0.14.1)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)